_pages/en/saas-and-cloud-strategy.md

```diff
@@ -0,0 +1,230 @@
+---
+layout: default
+title: SaaS vs Cloud Strategy
+lang: en
+ref: documents
+status: posted
+permalink: /saas-vs-cloud.html
+---
+
+## Context
+
+The process of creating this strategy begun due to the team receiving a request to help answer the question as to whether one should procure a "Cloud-based SaaS Solution" or a "DevOps In-house Build Cloud deployed solution." This document is attempting to assist in clarifying the terms used in the question, and help guide this decision, and similar decisions that will need to be made going forward, by defining the strategy which will clarify how and why each technology should be used to assist the organization in better delivering on its mandate.
+
+## The Question of Cloud
```

Comment:
I like the consistency of subsections for both "cloud" and "SaaS" main sections. But I think they may be better reframed as such:

- Driver: what's pushing us to change?
This is not just policy requirements. It's also that drive for agility and speed that you talk about.
- Desired State
---- Outcomes (instead of benefits)
- Strategy
--- Guiding Policy 
--- Immediate actions

The current wording in the "immediate actions" contain both policy and actions. Policy frame decision making like actions that such decisions do. But we go further than policy, we also prescribe actions.

Reply:
Instead of "Question of Cloud" and "Question of SaaS" use

"When to use Cloud" and "When to use SaaS"